src/pages/VoucherRedemption/Landing.tsx

```diff
@@ -40,7 +40,12 @@ const LandingCard = (props: Props) => {
         <br />
       </CardContainer>
       <br />
-      <Button color="#ab192e">{voucher.locations[0]}</Button>;
+      <Button color='#ab192e'>
+        {voucher.location.line1}
```

Comment:
are you sure that we only have a single location?

Reply:
I thought about this a bit. All of our merchants right now only have one location, and a lot of the UI in the sellers page and redemption flow is predicated on there only being one location. Given our target merchant demographic (mom and pop stores), I think its safe to assume that there will only be one location for now. If that changes in the future, we can re-evaluate the product experience.

Reply:
I agree here. I think we should amend the API to only return a single location, and in parallel make the frontend handle only that location as well

Reply:
https://github.com/sendchinatownlove/ruby/issues/157

Reply:
https://github.com/sendchinatownlove/sendchinatownlove.github.io/issues/185